frontend/views/containers/dashboard/GroupMembersAllModal.vue

```diff
@@ -1,18 +1,227 @@
+import { OPEN_MODAL } from '@utils/events.js'
 import ModalBaseTemplate from '@components/modal/ModalBaseTemplate.vue'
+import UserImage from '@components/UserImage.vue'
+import GroupMemberMenu from '@containers/dashboard/GroupMemberMenu.vue'
 
 export default {
-  name: 'GroupMembersAllModal',
+  name: 'IncomeDetails',
   components: {
-    ModalBaseTemplate
+    ModalBaseTemplate,
+    UserImage,
+    GroupMemberMenu
+  },
+  data () {
+    return {
+      form: {
+        search: null
+      }
+    }
   },
-  props: {
-    name: String
+  computed: {
+    ...mapGetters([
+      'groupProfiles',
+      'globalProfile',
+      'groupMembersCount'
+    ]),
+    searchResult () {
+      return Object.keys(this.groupProfiles)
+        .map(username => {
+          const inList = (n) => n.toUpperCase().indexOf(this.form.search.toUpperCase()) > -1
+          const { name, displayName } = this.globalProfile(username)
+          if (!this.form.search || inList(name) || (displayName ? inList(displayName) : false)) {
+            return { username, name, displayName }
+          }
+        })
+        .filter(profile => profile, [])
+    }
+  },
+  methods: {
+    openModal (name) {
+      sbp('okTurtles.events/emit', OPEN_MODAL, name)
+    },
+    closeModal () {
+      this.$refs.modal.close()
+    },
+    toChat () {
+      this.$router.push({ path: '/chat' })
+    }
   }
 }
 </script>
+
+<style lang="scss" scoped>
+@import "@assets/style/_variables.scss";
+.c-container {
+  height: 100%;
+  width: 100%;
+  background-color: $general_2;
+}
+
+.c-header,
+.c-container {
+  @include tablet {
+    width: 38.75rem;
+    max-width: 100%;
+  }
+}
+
+.c-header {
+  display: flex;
+  height: 4.75rem;
+  justify-content: center;
+  align-items: center;
+  padding-top: 0;
+  background-color: $background_0;
+  margin: 0 -1rem;
+
+  @include tablet {
+    padding-top: $spacer-lg;
+    justify-content: flex-start;
+    background-color: transparent;
+    margin: 0;
+  }
+}
+
+.c-card {
+  margin-top: 1.5rem;
+}
+
+.c-member-count {
+  margin-top: .5rem;
+  margin-bottom: 1.5rem;
+}
+
+.input-combo .is-icon {
+  left: 0;
+  right: auto;
+}
+
+.c-table {
+  width: 100%;
+  tr {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+}
+
+.c-identity {
+  display: flex;
+  align-items: center;
+}
+
+.c-name {
+  margin-left: 1.5rem;
+  min-width: 100%;
+}
+
+.c-display-name {
+  color: var(--text_1);
+}
+
+.c-group-member {
+  height: 4.5rem;
+  padding: 0;
+  transition: opacity ease-in .25s, height ease-in .25s;
+}
+
+.slide-list-enter, .slide-list-leave-to {
+  height: 0;
+  opacity: 0;
+}
+
+.slide-list-leave-active {
+  overflow: hidden;
+  border-bottom: 0;
+}
+
+.c-actions-buttons {
+  display: none;
+
+  i {
+    margin-right: .5rem;
+  }
+
+  @include tablet {
+    display: block;
+  }
+}
+
+.c-menu {
+  @include tablet {
+    display: none;
+  }
+}
+
+.button + .button {
+  margin-left: .5rem;
```

Comment:
we have the class `.buttons` for this same purpose. In this case we just need to remove the `margin-top` that comes from it.